avatar.css

```diff
@@ -0,0 +1,295 @@
+
+
+
+
+  .span-detail{
+    color:var(--avatar-font-color);
+    letter-spacing:1px;
+    display:flex;
+    justify-content:space-around;
+  }
+  
+  .flex-center{
+    display: flex;
+    height: 100vh;
+    width: 100vw;
+    justify-content:space-around;
+    align-content: center;
+    flex-wrap: wrap;
+    gap:50px;
+}
+  
+.avatar-s-x{
+  width:var(--avatar-s-x);
+  height:var(--avatar-s-x);
+}
+
+.avatar-md-x{
+  width:var(--avatar-md-x);
+  height:var(--avatar-md-x);
+}
+.avatar-lg-x{
+  width:var(--avatar-lg-x);
+  height:var(--avatar-lg-x);
+}
+  
+  .avatar {
+    overflow: hidden;
+    border-radius: 50%;
+    position: relative;
+    background-color: var(--bg-secondary-color);
+    border: 2px solid var(--bg-secondary-color);
+  }
+   .avatar img {
+      object-fit: cover;
+      width: 100%;
+      height: 100%;
+    }
+  
+  .avatar-group {
+    display: flex;
+    margin-bottom:20px;
+    justify-content:flex-start;
+    }
+
+    .flex-col{
+      flex-direction: column;
+      gap:20px;
+      
+    }
+    
+    .avatar:hover:not(:last-of-type) {
+         transform: translate(10px);
+        }
+    
+    .avatar{
+      margin-left: -20px;
+      transition: transform 0.3s ease;
+      cursor: pointer;
+      
+     
+    }
+    .avatar:hover:not(:last-of-type) {
+        transform: translate(-10px);
+     }
+    
+    .hidden-avatars { 
+      border-radius:50%;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin-left: 3px;
+      margin-right: 13px;
+      background-color:var(--bg-secondary-color);
+      color:var(--avatar-font-color);
+  }
+
+  /* avatar with white background */
+  .margin-left-50px{
+    margin-left:50px;
+  }
+  .avatar-circle-wht{
+    overflow: hidden;
+    border-radius: 50%;
+    position: relative;
+    border: 3px solid var(--green-color) ;
+    box-shadow: 0px 0px 8px 1px var(--green-color);
+  }
+
+  .avatar-circle-wht img{
+    object-fit: cover;
+      width: 100%;
+      height: 100%;
+      border-radius:50%;
+      border:3px solid var(--bg-color);
+  }
+
+  /* avatar rectangle with shadow */
+  .avatar-rect{
+    overflow: hidden;
+    border-radius:2px;
+    position: relative;
+    box-shadow: 0px 7px 9px 1px var(--bg-secondary-color);
+  }
+  
+  .avatar-rect img{
+    object-fit: cover;
+    width: 100%;
```

Comment:
` object-fit: cover;
    width: 100%;
    height: 100%;`

used in multiple classes. put these in one separate class and use it.